cockpit/gui/camera/window.py

```diff
@@ -112,12 +112,14 @@ def onSaveSettings(self, settings):
 
     ## Load our settings from the provided dict.
     def onLoadSettings(self, settings):
-        for view in self.views:
-            view.disableCamera()
-        for i, camName in enumerate(settings.get('camera view window', [])):
-            camera = depot.getHandlerWithName(camName)
-            self.views[i].enableCamera(camera)
-
+        # This code breaks as the views dont nessecarily exist.
+        # now subscribing from individual cameras.
```

Comment:
does this mean the code should be removed then? Instead of commented out?

Reply:
Probably but there is one side effect of my solution, the channels may be activated in a different order and then the resulting files will have different image order.